src/python/fuzzing/strategy.py

```diff
@@ -72,6 +76,7 @@
 LIBFUZZER_STRATEGIES_WITH_PREFIX_VALUE = [
     CORPUS_SUBSET_STRATEGY,
     FORK_STRATEGY,
+    PEACH_GRAMMAR_MUTATION_STRATEGY,
```

Comment:
I don't think this will work as expected unfortunately. 

This (https://github.com/google/clusterfuzz/blob/5e9b02204428cb1a81993d512844966d2bee0a80/src/python/bot/fuzzers/libFuzzer/stats.py#L148) assumes that the part after the prefix is an int, which is not the case here.

We need to do some refactoring to support this. While the ideal refactor is a bit complicated (I can do this later), to unblock you you can probably just refactor LIBFUZZER_STRATEGIES_WITH_PREFIX_VALUE such that it's a dict where the values indicate the type.

Reply:
Changing LIBFUZZER_STRATEGIES_WITH_PREFIX_VALUE to a dict is not going to work because all that is available is the strategy.name, not the strategy itself. Instead I can make another dict that maps strategy.name -> type. Is that sufficient?

Reply:
That also works.

Reply:
Done